Makefile.js

```diff
@@ -42,29 +42,30 @@ target.all = function() {
 };
 
 target.lint = function() {
+    const flag = process.argv[process.argv.length - 1] === "--fix" ? "--fix" : "";
```

Comment:
I think this is certainly better than duplicating all the logic, but it does feel kind of brittle. Could we just all arguments through?

Reply:
> Could we just all arguments through?

I didn't get this.  
you meant passing all arguments from `process.argv` ?

Reply:
That's right :+1:

Reply:
done 👍